packages/teko/tests/unit_tests/tStratimikosFactory.cpp

```diff
@@ -256,7 +258,7 @@ const RCP<Tpetra::Operator<ST,LO,GO,NT> > buildStridedSystem(const Teuchos::RCP<
         vPtr = (n==0) ? valuesA : valuesB;
         iPtr = indices;
      
-        if(gid<numUnks) {
+        if(gid<(GO)numUnks) {
```

Comment:
Prefer C++ - style casts to this C-style cast.

Reply:
Do you mean: ```reinterpret_cast<GO>(numUnks)``` instead of ```(GO)numUnks```?  I get a compilation error when I try this, hmmm...

Reply:
`static_cast<GO>(numUnks)` or just `GO(numUnks)`.

Reply:
I prefer the C-style syntax, I guess... But I'm ok with the change - just made it.